pkg/registry/apis/playlist/register.go

```diff
@@ -32,18 +33,21 @@ type PlaylistAPIBuilder struct {
 	namespacer request.NamespaceMapper
 	gv         schema.GroupVersion
 	features   featuremgmt.FeatureToggles
+	kvStore    *kvstore.NamespacedKVStore
 }
 
 func RegisterAPIService(p playlistsvc.Service,
```

Comment:
Question: have you checked if we can modify this signature to have the first argument be a `context.Context`? I think it would require regenerating the wire files. In case it doesn't work:

1. It may be because we don't yet have a wire provider return a `context.Context`. We could try to add a wire provider that provides a `context.Context`, simply returning `context.Background()` for now.
2. In the worst case (e.g. if the above is too much work, or unrelated, etc.), we could just pass `context.Background()` to `SetDualWritingMode` here.

The idea is that we push the need for a proper `context.Context` as far above in dependencies as possible, so that when we take the step to properly providing a `context.Context` the refactor is not so big.

Reply:
Thanks for the review! That makes sense about pushing the need for the context.Context up. Ended up going with option 2 for now.